endpoints/index.md

```diff
@@ -46,7 +46,7 @@ time.
 
  * As we previously explained [here][index-pulling], clients MAY use the
    `index` and `get` endpoints as a pull-based method of synchronization,
-   alternative (or rather complementary) to CNRs. It is RECOMMENDED for the
+   complementary to CNRs. It is RECOMMENDED for the
```

Comment:
is not clear where is written that the CNR is mandatory as is written in the comment of the commit?
is just written to use index and get as other ways complementary to CNR...

Reply:
CNR usage is globally mandatory. We changed `manifest-entry.xsd` files to reflect that.

This comment in the index API had to be updated, as pull-based synchronization cannot be treated as an alternative to CNRs anymore.